### PR TITLE
Remove adjustment state

### PR DESCRIPTION
**Pull request by mbj**

### CODE REVIEW

File: backend/app/views/spree/admin/adjustments/_adjustment.html.erb [hunk 1]
```diff
@@ -6,11 +6,8 @@
...
   <td class="align-center"><%= adjustment.label %></td>
   <td class="align-center"><%= adjustment.display_amount.to_html %></td>
-  <td class="align-center"><%= adjustment_state(adjustment) %></td>
   <td class='actions'>
-    <% if adjustment.open? %>
-      <%= link_to_edit adjustment, :no_text => true %>
-      <%= link_to_delete adjustment, :no_text => true %>
-    <% end %>
+    <%= link_to_edit adjustment, :no_text => true %>
+    <%= link_to_delete adjustment, :no_text => true %>
```
Comment (dkubb) [inline backend/app/views/spree/admin/adjustments/_adjustment.html.erb]: This should use ruby 1.9 style hashes.

File: core/app/models/spree/adjustment.rb [hunk 2]
```diff
@@ -75,19 +65,13 @@ def adjustables(adjustables)
       memory_scope(:credit)  { |adjustment| adjustment.amount <  0   }
       memory_scope(:nonzero) { |adjustment| !adjustment.amount.zero? }
 
-      memory_scope_attribute_value(:open,        :state,           'open'                  )
```
Comment (dkubb) [inline core/app/models/spree/adjustment.rb]: The closing `)` in this block can be moved left.

**Comment by mbj** [comment]
@dkubb This is ready for review. Next iterations can split `Spree::Adjustment` into `Spree::LineItemAdjustment` and `Spree::OrderAdjustment` reducing lots of redundancy.

**Comment by dkubb** [comment]
@mbj given the removal of the `nonzero` scope, should we add a database constraint on the `mrh-spree` side that says the adjustments cannot be `$0`? I realize this will require us to also delete any existing ones that are `$0`, but I think that's ok.

**Comment by mbj** [comment]
> @mbj given the removal of the nonzero scope, should we add a database constraint on the mrh-spree side that says the adjustments cannot be $0? I realize this will require us to also delete any existing ones that are $0, but I think that's ok.

Yes. Same for non eligible adjustments, which come in significant numbers in our case.

**Comment by dkubb** [comment]
> Yes. Same for non eligible adjustments, which come in significant numbers in our case.

I can't wait for this removal. So many invalid states will be removed.

**Pull request merged**